### PR TITLE
add: snackbar typography lists usage and refactor

### PR DESCRIPTION
**Pull request by PuneetP16**
## Usage
image

## Typography
image

### Menu List
image
### Stacked List
image

## Snackbar
image

### CODE REVIEW

File: src/components/typography/typography.html [hunk 1]
```diff
@@ -0,0 +1,197 @@
...
+						@import url("https://magnificentui.netlify.app/src/components/typography/typography.css");
+	
+						// HTML
+						<link rel="stylesheet" href="https://magnificentui.netlify.app/src/components/typography/typography.css">
+					</script>
+				</div>
+	
+				<!-- Headings component -->
+				<h3>Headings</h3>
+
```
Comment (deekshithmd) [inline src/components/typography/typography.html:118]: try to reduce these line gaps

File: src/components/snackbar/snackbar.css [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+	opacity: 1;
+}
+
+.snackbar .btn--contained {
+	width: 6.4rem;
+	padding: 0 0.8rem;
+}
+
+.snackbar .btn--close--transparent {
+	margin-left: 0.8rem;
```
Comment (deekshithmd) [inline src/components/snackbar/snackbar.css]: better to use px for values less than 10px and also for margin and padding

File: src/components/lists/lists.css [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+	padding: 2px 8px;
+	width: 100%;
+	border-radius: 8px;
+    color: var(--COLOR-BLACK);
+}
+
+
+.lists__links:hover {
+    color: var(--COLOR-PRIMARY);
+    background-color: rgb(236, 253, 255);
```
Comment (deekshithmd) [inline src/components/lists/lists.css]: using color variables will be good

File: src/components/lists/lists.css [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+.lists__spaced {
+    gap: 3px;
+}
+.lists--stacked{
+    gap:0;
+}
+.lists__head {
+	font-weight: 500;
+	font-size: 2.2rem;
+
```
Comment (deekshithmd) [inline src/components/lists/lists.css]: avoid these line gaps and give line gap between classes

**Review by deekshithmd — APPROVED** [review]

File: src/components/lists/lists.css (hunk 4 above)
Comment (PuneetP16) [reply src/components/lists/lists.css]: Thanks for reviewing and mentioning changes, will update it all.

File: src/components/typography/typography.html (hunk 1 above)
Comment (PuneetP16) [reply src/components/typography/typography.html:118]: Cool! I have kept them deliberately to make them more readable. I will trim them after completing the project.

File: src/components/snackbar/snackbar.css (hunk 2 above)
Comment (PuneetP16) [reply src/components/snackbar/snackbar.css]: Done!

File: src/components/lists/lists.css (hunk 3 above)
Comment (PuneetP16) [reply src/components/lists/lists.css]: Done

File: src/components/button/button.css [hunk 5]
```diff
@@ -28,6 +28,12 @@
 	opacity: 1;
 }
 
+.btn--contained {
+	background: transparent;
+    border:none;
+	box-shadow: none;
+}
```
Comment (bhushannaikaroskar) [inline src/components/button/button.css]: align code properly

File: src/components/card/card.css [hunk 6]
```diff
@@ -88,22 +75,56 @@
 	opacity: 1;
 }
 
-/* ------WITH TEXT-OVERLAY-------- */
-.card {
-	position: relative;
+.card__actions button {
+	padding: 0.5rem 0.6rem;
 }
```
Comment (bhushannaikaroskar) [inline src/components/card/card.css]: values should have been in px right?

File: src/components/card/card.css [hunk 7]
```diff
@@ -138,3 +155,110 @@
...
+
+.ribbon {
+	position: absolute;
+	right: -5px;
+	top: -5px;
+	z-index: 1;
+	overflow: hidden;
+	width: 75px;
+	height: 75px;
+	text-align: right;
```
Comment (bhushannaikaroskar) [inline src/components/card/card.css]: convert to rem

File: src/components/card/card.css [hunk 8]
```diff
@@ -138,3 +155,110 @@
...
+	transform: rotate(45deg);
+	-webkit-transform: rotate(45deg);
+	width: 100px;
+	display: block;
+	background: #79a70a;
+	background: linear-gradient(#2989d8 0%, #1e5799 100%);
+	box-shadow: 0 3px 10px -5px rgba(0, 0, 0, 1);
+	position: absolute;
+	top: 19px;
+	right: -21px;
```
Comment (bhushannaikaroskar) [inline src/components/card/card.css]: same here rem conversion needed

**Review by bhushannaikaroskar — COMMENTED** [review]
I think on multiple places used px and instead of rem need to change that

File: src/components/button/button.css (hunk 5 above)
Comment (PuneetP16) [reply src/components/button/button.css]: Done thanks for review

File: src/components/card/card.css (hunk 6 above)
Comment (PuneetP16) [reply src/components/card/card.css]: yeah! will update.

File: src/components/card/card.css (hunk 7 above)
Comment (PuneetP16) [reply src/components/card/card.css]: Thanks for mentioning it. I missed this.

File: src/components/card/card.css (hunk 8 above)
Comment (PuneetP16) [reply src/components/card/card.css]: I have updated all the changes you suggested.

**Comment by sushmita12345** [comment]
Suggest to do proper code alignment.
Good to merge.

**Review by sushmita12345 — APPROVED** [review]

**Comment by PuneetP16** [comment]
> 

Thanks for PR review.

File: src/styles/dark-colors.css [hunk 9]
```diff
@@ -1,47 +1,47 @@
...
+	--dark-text--COLOR-STRINGS: #a9f0ff;
+	--dark-text--COLOR-IMPORT: #ff941a;
+	--dark-bg--COLOR-BOX-SHADOW1: rgba(0, 0, 0, 0.2);
+	--dark-bg--COLOR-BOX-SHADOW2: rgba(0, 0, 0, 0.14);
+	--dark-bg--COLOR-BOX-SHADOW3: rgba(0, 0, 0, 0.12);
+	--dark-bg--COLOR-BOX-SHADOW4: rgba(40, 40, 74, 0.1);
+	--dark-bg--COLOR-BOX-SHADOW5: rgba(0, 0, 0, 0.08);
+	--dark-text--COLOR-ACTIVE: #1affff;
+	--dark-bg--COLOR-DISCOUNT-BG: #513100;
+	--dark-bg--COLOR-CARD-BOX-SHADOW: rgba(0, 0, 0, 0.56);
```
Comment (m26unkwn) [inline src/styles/dark-colors.css:46]: This is call good code structure using variable very well.

**Review by m26unkwn — APPROVED** [review]
Love your code and your new UI colors.

**Review by bhushannaikaroskar — APPROVED** [review]
Approving the request after changes. I would suggest you to use vscode extension prettier. It takes care of spacing and alignment on saving the file. Do use it it would save your time a lot.

File: src/components/typography/typography.html (hunk 1 above)
Comment (deekshithmd) [reply src/components/typography/typography.html:118]: ok

**Review by dhruvin25799 — APPROVED** [review]

File: src/styles/dark-colors.css (hunk 9 above)
Comment (PuneetP16) [reply src/styles/dark-colors.css:46]: Thanks, but will apply dark theme in next draft.

**Pull request merged**